.github/workflows/ci.yaml

```diff
@@ -130,14 +144,26 @@ jobs:
       env:
         TAG: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}@${{ env.NETWORK_OPERATOR_DIGEST }}
         BUNDLE_IMG: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}-bundle:${{ env.VERSION_WITH_PREFIX }}
-        VERSION: ${{ env.VERSION_WITHOUT_PREFIX }}
         NGC_CLI_API_KEY: ${{ secrets.NVCR_TOKEN }}
       run: |
-        version_major_minor=$(echo $VERSION_WITH_PREFIX | grep -Eo 'v[0-9]+\.[0-9]+')
-        export CHANNELS=stable,$version_major_minor
-        export DEFAULT_CHANNEL=$version_major_minor
+        if [[ "$is_push" == "false" ]]; then
+          export VERSION=${{ env.VERSION_WITHOUT_PREFIX }}
+          version_major_minor=$(echo $VERSION_WITH_PREFIX | grep -Eo 'v[0-9]+\.[0-9]+')
+          export CHANNELS=stable,$version_major_minor
+          export DEFAULT_CHANNEL=$version_major_minor
+        else
+          export DEFAULT_CHANNEL=v1.1 # hard coded
```

Comment:
it should be:
export DEFAULT_CHANNEL=v1.1,$VERSION
export CHANNELS=stable,$VERSION